assets/scss/templates/_reportes.scss

```diff
@@ -225,6 +225,7 @@
   max-width: $dp-wrapper-max-width;
   margin: $dp-spaces--lv4 auto;
   font-family: $dp-font-family;
+  position: relative;
```

Comment:
Is that enough? does it not break anything?

Reply:
its necessary because the spinner has absolute position and need parent with relative to align inside the ranking box